common/components/constants/ProjectConstants.js

```diff
@@ -1,3 +1,4 @@
+// TODO: Delete?
```

Comment:
Confirm if we can delete this?

Reply:
The in-progress Groups code is still using it, so adding a note to delete it after that feature is finished.